public/app/plugins/panel/geomap/editor/StyleRuleEditor.tsx

```diff
@@ -86,30 +115,39 @@ export const StyleRuleEditor: FC<StandardEditorProps<FeatureStyleConfig, any, an
     <div className={styles.rule}>
       <InlineFieldRow className={styles.row}>
         <InlineField label="Rule" labelWidth={LABEL_WIDTH} grow={true}>
-          <Input
-            type="text"
+          <Select
+            menuShouldPortal
             placeholder={'Feature property'}
             value={`${value?.rule?.property}`}
-            onChange={onChangeComparisonProperty}
+            options={propertyOptions}
+            onChange={onChangeProperty}
```

Comment:
can you add:
`isClearable` and `allowCreateOption`  (the real option is close to that, but AFK)

Reply:
it will allow typing any value even if not in your select